File: modul 5/python/PRAK503-2410817110008-MuhammadAzmaAlFaqih.py
def maksimal(maks, nilai):
    return maks if maks>nilai else nilai

def minimal(minim, nilai):
    return minim if minim<nilai else nilai

maks = -100000
minim = 10000
inputs = []
batas = 0
nilai = 0
bilangan = int(input())
while len(inputs) < bilangan:
    nilai = input()
    _nilai = nilai.split()
    
    for i in _nilai:
        inputs.append(int(i))

for i in inputs:
    maks = maksimal(maks,i)
    minim = minimal(minim,i)

print(maks, minim)

File: modul 5/c/PRAK503-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>

int maksimal(int a, int b){
  return (b>a) ? b : a;
}
int minimal(int a, int b){
  return (b<a) ? b : a;
}

int main(){
  int batas = 0;
  int maks = -10000;
  int minim = 10000;
  int bilangan;
  scanf("%d", &bilangan);
  while(batas<bilangan){
    int nilai;
    scanf("%d",&nilai);
    maks = maksimal(maks,nilai);
    minim = minimal(minim,nilai);
    batas++;
  }
  printf("%d %d\n", maks, minim);
}